### ISSUE TITLE
Issue about the performance

### ISSUE THREAD

**Issue by coldmanck** [ruotianluo/self-critical.pytorch#32]
Hi @ruotianluo 

Thank you for your fantastic work. I followed your implementation to train the captioning model (TopDown) with the top 36 features I downloaded from the author. And below is the result:
<img width="1008" alt="screen shot 2018-01-31 at 8 30 40 pm" src="https://user-images.githubusercontent.com/7726203/35623210-b0a9a522-06c5-11e8-9826-b0a601489918.png">
(The second row result is took from your README and the third row is my result.)
Do you have any idea why the performance still can not reach the reported standard?

**Comment by ruotianluo** [comment]
Did you match the hidden size?   Are you using the same batch size? Use 20-100 instead of 36.

**Comment by coldmanck** [comment]
Yes I found I did not follow the same hyper parameters (hidden size, batch_size, learning rate decay). I will fix them and try again. Thanks very much! :)

**Comment by loriqing** [comment]
> Yes I found I did not follow the same hyper parameters (hidden size, batch_size, learning rate decay). I will fix them and try again. Thanks very much! :)

hello, what's the performance of your new try? I also encountered this problem.

**Comment by jamiechoi1995** [comment]
@coldmanck 
may I ask if "w/o bottom-up features" means using 14*14*2048 features extracted from resner101?
